ajax/libs/Sortable/package.json

```diff
@@ -1,13 +1,12 @@
 {
   "name": "Sortable",
   "title": "Sortable",
-  "version": "1.6.0",
```

Comment:
Please do not manually touch the version info here.

Reply:
Sorry about that; still learning how this works.